API.md

```diff
@@ -0,0 +1,552 @@
+# API
+
+* [Static Functions](#static-functions)
+  * [`create`](#create)
+* [Instance Methods](#instance-methods)
+  * [`start`](#start)
+  * [`stop`](#stop)
+  * [`dial`](#dial)
+  * [`dialProtocol`](#dialProtocol)
+  * [`hangUp`](#hangUp)
+  * [`handle`](#handle)
+  * [`unhandle`](#unhandle)
+  * [`peerRouting.findPeer`](#peerRouting.findPeer)
+  * [`contentRouting.findProviders`](#contentRouting.findProviders)
+  * [`contentRouting.provide`](#contentRouting.provide)
+  * [`_dht.put`](#_dht.put)
+  * [`_dht.get`](#_dht.get)
+  * [`_dht.getMany`](#_dht.getMany)
+  * [`pubsub.getPeersSubscribed`](#pubsub.getPeersSubscribed)
+  * [`pubsub.getTopics`](#pubsub.getTopics)
+  * [`pubsub.publish`](#pubsub.publish)
+  * [`pubsub.subscribe`](#pubsub.subscribe)
+
+## Static Functions
+
+### create
+
+Creates an instance of Libp2p.
+
+`create(options)`
+
+#### Parameters
+
+| Name | Type | Description |
+|------|------|-------------|
+| options | `Object` | libp2p options |
```

Comment:
What are the options?

Reply:
I would like to have a specific MD doc for the options/libp2p configuration, where this would point out. As there are so many options and instructions for configuration, I think it will be easier for people reading the docs to digest a lighter API, while being able to consult the options when needed. What do you think?

Reply:
I will create that in a new PR next week

Reply:
So, after writting that in #495 , I think I will document the first level properties of the options object here and point to the other doc